systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -95,9 +95,17 @@ def get_control_if(self):
                 return ip
         return None
 
+    def check_if_link_up(self,dev):
+        cmd="ip link show dev %s | tr '\n' ' ' | cut -d ' ' -f 9"%dev
```

Comment:
Indeed and also use the -o flag for 'ip':

 ip -o link show dev <dev>

Then split the string using Python.